change/@azure-msal-common-7f106e66-8f0f-4bfe-887f-3199473210de.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "major",
```

Looks like we need this to be a major bump instead.